cvat-canvas3d/src/typescript/canvas3d.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import pjson from '../../package.json';
+import '../scss/canvas.scss';
+import { Canvas3dController, Canvas3dControllerImpl } from './canvas3dController';
+import { Canvas3dModel, Canvas3dModelImpl, Mode } from './canvas3dModel';
+import { Canvas3dView, Canvas3dViewImpl } from './canvas3dView';
+import { Master } from './master';
+
+const Canvas3dVersion = pjson.version;
+
+interface Canvas3d {
+    html(): any;
+    setup(frameData: any, objectStates: any[], zLayer?: number): void;
+    isAbleToChangeFrame(): boolean;
```

Comment:
The method is not presented in README.md as an interface method.

Reply:
Resolved